src/sonic-ctrmgrd/tests/docker.py

```diff
@@ -0,0 +1,7 @@
+#! /usr/bin/python3
+
+def from_env():
+    print("docker.py from_env called")
+    return None
```

Comment:
Is it necessary to have a file just for this function?

Reply:
Yes, because we don't have docker.py inside slave docker

Reply:
This seems a bit hacky. Can you simply mock it instead? Something like:

```python3
mock_docker = MagicMock()
sys.modules["docker"] = mock_docker
```

Reply:
Done

Reply:
This is better and I'm OK with this solution, but I wonder if we even need this file at all, since the function is essentially a no-op, it just prints a message. If the message is used to determine the function was called, you should simply be able to check that the function was called via the `MagicMock` object. We could save this for future clean-up/refactoring.